demo/qat-init.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# shellcheck disable=SC2154
```

Comment:
why?

Reply:
If you are saying about the shellcheck disable, that seems necessary since.. we use the ENABLING_PF_PCI_IDs in the script that is not assigned in the script..
I actually tried the variable have empty value by doing

> ENABLING_PF_PCI_IDs=${ENABLING_PF_PCI_IDs:-}

that works when just running the script in linux, but in toybox it does not.

Reply:
> This works when running the script in linux, but it does not work in toybox.

This worked OK:
``` 
$ docker run -it --entrypoint /bin/sh --env FOO="4140" intel/intel-qat-initcontainer:devel 
# BAR=${FOO:-"3c79 4140"}
# set|grep BAR
BAR=4140
$ docker run -it --entrypoint /bin/sh  intel/intel-qat-initcontainer:devel 
# BAR=${FOO:-"3c79 4140"}
# set|grep BAR
BAR="3c79 4140"
```

Reply:
Thanks!